chatminer/visualizations.py

```diff
@@ -387,3 +388,20 @@ def is_radar_registered():
     except ValueError:
         return False
     return True
+
+def get_emojis(input_string,append_emoji_list):
+    for i in range(0,len(input_string)):
+        if emoji.is_emoji(input_string[i]):
+            append_emoji_list.append(emoji.demojize(input_string[i]))
+
+def piechart_emoji(
+    df,
+    n_largest=10
+):
+    emoji_list=[]
+    for i in range(0,len(df.index)):
+        get_emojis(df["message"].iloc[i],emoji_list)
+    emoji_list = pd.Series(list(emoji_list))
+    freqs = emoji_list.value_counts()
+    freqs=freqs.nlargest(n=n_largest)
+    return freqs.plot.pie()
```

Comment:
The other visualizations provide the option to hand over the matplotlib axis as a parameter (`ax=None`) and fallback to a default axis via
```
if ax is None:
        _, ax = plt.subplots()
```
I would see this for the piechart as well to be well aligned to the other plots.